src/content/certifications/exam-faqs/vault-operations-professional.mdx

```diff
@@ -11,7 +11,15 @@ We strongly recommend passing the associate-level Vault exam before taking the p
 - Information security fundamentals such as network security and RBAC
 - Understand the concepts and functionality of infrastructure running in containers including starting and stopping services, and reading logs
 
-![](https://www.datocms-assets.com/2885/1671559871-vault-badge-image-export.svg)
+<ThemedImage
```

Comment:
Interestingly, there is a 'backslash' being inserted by the markdown processor and I can't figure out why. @zchsh do you have any hot tips? 

<img width="701" alt="Screenshot 2023-03-28 at 3 14 43 PM" src="https://user-images.githubusercontent.com/36613477/228378622-7195200a-8705-4314-878c-71959ea31722.png">

Reply:
Ooh I'm not sure 🤔 Sounds like Bryce had a good lead in Slack here re: Certifications not supporting MDX. Happy to dive in deeper if that path doesn't yield much 🙇‍♂️

Reply:
I ended up working around this for now by not using objects in the props. I still think there is an issue with the MDX, but I opted for just getting this in with the workaround instead of going down the wormhole there and added a comment as a breadcrumb.